client/package.json

```diff
@@ -2,18 +2,18 @@
   "name": "sqflint",
   "displayName": "SQFLint",
   "description": "Parse errors, hover help for functions/commands and more.",
-  "author": "Jan Zípek",
+  "author": "Jan Zípek, Senfo",
   "icon": "images/sqflint.png",
   "license": "MIT",
-  "version": "0.10.1",
-  "publisher": "skacekachna",
+  "version": "0.12.0",
+  "publisher": "senfo",
```

Comment:
I think this should also refer to this repo instead of you fork, if I were to merge this